<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>520</width>
    <height>291</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="contextMenuPolicy">
   <enum>Qt::PreventContextMenu</enum>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="toolTipDuration">
   <number>0</number>
  </property>
  <property name="layoutDirection">
   <enum>Qt::LeftToRight</enum>
  </property>
  <property name="styleSheet">
   <string notr="true">/*Copyright (c) DevSec Studio. All rights reserved.

MIT License

Permission is hereby granted, free of charge, to any person obtaining a copy
of this software and associated documentation files (the &quot;Software&quot;), to deal
in the Software without restriction, including without limitation the rights
to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
copies of the Software, and to permit persons to whom the Software is
furnished to do so, subject to the following conditions:

The above copyright notice and this permission notice shall be included in all
copies or substantial portions of the Software.

THE SOFTWARE IS PROVIDED *AS IS*, WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE SOFTWARE.
*/

/*-----QWidget-----*/
QWidget
{
	background-color: #3a3a3a;
	color: #fff;
	selection-background-color: #b78620;
	selection-color: #000;

}


/*-----QLabel-----*/
QLabel
{
	background-color: transparent;
	color: #fff;

}


/*-----QMenuBar-----*/
QMenuBar 
{
	background-color: qlineargradient(spread:repeat, x1:1, y1:0, x2:1, y2:1, stop:0 rgba(57, 57, 57, 255),stop:1 rgba(50, 50, 50, 255));
	border: 1px solid #000;
	color: #fff;

}


QMenuBar::item 
{
	background-color: transparent;

}


QMenuBar::item:selected 
{
	background-color: rgba(183, 134, 32, 20%);
	border: 1px solid #b78620;
	color: #fff;

}


QMenuBar::item:pressed 
{
	background-color: rgb(183, 134, 32);
	border: 1px solid #b78620;
	color: #fff;

}


/*-----QMenu-----*/
QMenu
{
    background-color: qlineargradient(spread:repeat, x1:1, y1:0, x2:1, y2:1, stop:0 rgba(57, 57, 57, 255),stop:1 rgba(50, 50, 50, 255));
    border: 1px solid #222;
    padding: 4px;
	color: #fff;

}


QMenu::item
{
    background-color: transparent;
    padding: 2px 20px 2px 20px;

}


QMenu::separator
{
   	background-color: rgb(183, 134, 32);
	height: 1px;

}


QMenu::item:disabled
{
    color: #555;
    background-color: transparent;
    padding: 2px 20px 2px 20px;

}


QMenu::item:selected
{
	background-color: rgba(183, 134, 32, 20%);
	border: 1px solid #b78620;
	color: #fff;

}


/*-----QToolBar-----*/
QToolBar
{
	background-color: qlineargradient(spread:repeat, x1:1, y1:0, x2:1, y2:1, stop:0 rgba(69, 69, 69, 255),stop:1 rgba(58, 58, 58, 255));
	border-top: none;
	border-bottom: 1px solid #4f4f4f;
	border-left: 1px solid #4f4f4f;
	border-right: 1px solid #4f4f4f;

}


QToolBar::separator
{
	background-color: #2e2e2e;
	width: 1px;

}


/*-----QToolButton-----*/
QToolButton 
{
	background-color: transparent;
	color: #fff;
	padding: 5px;
	padding-left: 8px;
	padding-right: 8px;
	margin-left: 1px;
}


QToolButton:hover
{
	background-color: rgba(183, 134, 32, 20%);
	border: 1px solid #b78620;
	color: #fff;
	
}


QToolButton:pressed
{
	background-color: qlineargradient(spread:repeat, x1:1, y1:0, x2:1, y2:1, stop:0 rgba(57, 57, 57, 255),stop:1 rgba(50, 50, 50, 255));
	border: 1px solid #b78620;

}


QToolButton:checked
{
	background-color: qlineargradient(spread:repeat, x1:1, y1:0, x2:1, y2:1, stop:0 rgba(57, 57, 57, 255),stop:1 rgba(50, 50, 50, 255));
	border: 1px solid #222;
}


/*-----QPushButton-----*/
QPushButton
{
	background-color: qlineargradient(spread:repeat, x1:1, y1:0, x2:1, y2:1, stop:0 rgba(84, 84, 84, 255),stop:1 rgba(59, 59, 59, 255));
	color: #ffffff;
	min-width: 80px;
	border-style: solid;
	border-width: 1px;
	border-radius: 3px;
	border-color: #051a39;
	padding: 5px;

}


QPushButton::flat
{
	background-color: transparent;
	border: none;
	color: #fff;

}


QPushButton::disabled
{
	background-color: #404040;
	color: #656565;
	border-color: #051a39;

}


QPushButton::hover
{
	background-color: rgba(183, 134, 32, 20%);
	border: 1px solid #b78620;

}


QPushButton::pressed
{
	background-color: qlineargradient(spread:repeat, x1:1, y1:0, x2:1, y2:1, stop:0 rgba(74, 74, 74, 255),stop:1 rgba(49, 49, 49, 255));
	border: 1px solid #b78620;

}


QPushButton::checked
{
	background-color: qlineargradient(spread:repeat, x1:1, y1:0, x2:1, y2:1, stop:0 rgba(74, 74, 74, 255),stop:1 rgba(49, 49, 49, 255));
	border: 1px solid #222;

}


/*-----QLineEdit-----*/
QLineEdit
{
	background-color: #131313;
	color : #eee;
	border: 1px solid #343434;
	border-radius: 2px;
	padding: 3px;
	padding-left: 5px;

}


/*-----QPlainTExtEdit-----*/
QPlainTextEdit
{
	background-color: #131313;
	color : #eee;
	border: 1px solid #343434;
	border-radius: 2px;
	padding: 3px;
	padding-left: 5px;

}


/*-----QTabBar-----*/
QTabBar::tab
{
	background-color: qlineargradient(spread:repeat, x1:1, y1:0, x2:1, y2:1, stop:0 rgba(84, 84, 84, 255),stop:1 rgba(59, 59, 59, 255));
	color: #ffffff;
	border-style: solid;
	border-width: 1px;
	border-color: #666;
	border-bottom: none;
	padding: 5px;
	padding-left: 15px;
	padding-right: 15px;

}


QTabWidget::pane 
{
	background-color: red;
	border: 1px solid #666;
	top: 1px;

}


QTabBar::tab:last
{
	margin-right: 0; 

}


QTabBar::tab:first:!selected
{
	background-color: #0c0c0d;
	margin-left: 0px;

}


QTabBar::tab:!selected
{
	color: #b1b1b1;
	border-bottom-style: solid;
	background-color: #0c0c0d;

}


QTabBar::tab:selected
{
	margin-bottom: 0px;

}


QTabBar::tab:!selected:hover
{
	border-top-color: #b78620;

}


/*-----QComboBox-----*/
QComboBox
{
    background-color: qlineargradient(spread:repeat, x1:1, y1:0, x2:1, y2:1, stop:0 rgba(84, 84, 84, 255),stop:1 rgba(59, 59, 59, 255));
    border: 1px solid #000;
    padding-left: 6px;
    color: #ffffff;
    height: 20px;

}


QComboBox::disabled
{
	background-color: #404040;
	color: #656565;
	border-color: #051a39;

}


QComboBox:on
{
    background-color: #b78620;
	color: #000;

}


QComboBox QAbstractItemView
{
    background-color: #383838;
    color: #ffffff;
    border: 1px solid black;
    selection-background-color: #b78620;
    outline: 0;

}


QComboBox::drop-down
{
	background-color: qlineargradient(spread:repeat, x1:1, y1:0, x2:1, y2:1, stop:0 rgba(57, 57, 57, 255),stop:1 rgba(50, 50, 50, 255));
    subcontrol-origin: padding;
    subcontrol-position: top right;
    width: 15px;
    border-left-width: 1px;
    border-left-color: black;
    border-left-style: solid; 

}


QComboBox::down-arrow
{
    image: url(://arrow-down.png);
    width: 8px;
    height: 8px;
}


/*-----QSpinBox &amp; QDateTimeEdit-----*/
QSpinBox,
QDateTimeEdit 
{
    background-color: #131313;
	color : #eee;
	border: 1px solid #343434;
	padding: 3px;
	padding-left: 5px;
    border-radius : 2px;

}


QSpinBox::up-button, 
QDateTimeEdit::up-button
{
	border-top-right-radius:2px;
	background-color: #777777;
    width: 16px; 
    border-width: 1px;

}


QSpinBox::up-button:hover, 
QDateTimeEdit::up-button:hover
{
	background-color: #585858;

}


QSpinBox::up-button:pressed, 
QDateTimeEdit::up-button:pressed
{
	background-color: #252525;
    width: 16px; 
    border-width: 1px;

}


QSpinBox::up-arrow,
QDateTimeEdit::up-arrow
{
    image: url(://arrow-up.png);
    width: 7px;
    height: 7px;

}


QSpinBox::down-button, 
QDateTimeEdit::down-button
{
	border-bottom-right-radius:2px;
	background-color: #777777;
    width: 16px; 
    border-width: 1px;

}


QSpinBox::down-button:hover, 
QDateTimeEdit::down-button:hover
{
	background-color: #585858;

}


QSpinBox::down-button:pressed, 
QDateTimeEdit::down-button:pressed
{
	background-color: #252525;
    width: 16px; 
    border-width: 1px;

}


QSpinBox::down-arrow,
QDateTimeEdit::down-arrow
{
    image: url(://arrow-down.png);
    width: 7px;
    height: 7px;

}


/*-----QGroupBox-----*/
QGroupBox 
{
    border: 1px solid;
    border-color: #666666;
	border-radius: 5px;
    margin-top: 20px;

}


QGroupBox::title  
{
    background-color: transparent;
    color: #eee;
    subcontrol-origin: margin;
    padding: 5px;
	border-top-left-radius: 3px;
	border-top-right-radius: 3px;

}


/*-----QHeaderView-----*/
QHeaderView::section
{
    background-color: qlineargradient(spread:repeat, x1:1, y1:0, x2:1, y2:1, stop:0 rgba(60, 60, 60, 255),stop:1 rgba(50, 50, 50, 255));
	border: 1px solid #000;
    color: #fff;
    text-align: left;
	padding: 4px;
	
}


QHeaderView::section:disabled
{
    background-color: #525251;
    color: #656565;

}


QHeaderView::section:checked
{
    background-color: qlineargradient(spread:repeat, x1:1, y1:0, x2:1, y2:1, stop:0 rgba(60, 60, 60, 255),stop:1 rgba(50, 50, 50, 255));
    color: #fff;

}


QHeaderView::section::vertical::first,
QHeaderView::section::vertical::only-one
{
    border-top: 1px solid #353635;

}


QHeaderView::section::vertical
{
    border-top: 1px solid #353635;

}


QHeaderView::section::horizontal::first,
QHeaderView::section::horizontal::only-one
{
    border-left: 1px solid #353635;

}


QHeaderView::section::horizontal
{
    border-left: 1px solid #353635;

}


QTableCornerButton::section
{
    background-color: qlineargradient(spread:repeat, x1:1, y1:0, x2:1, y2:1, stop:0 rgba(60, 60, 60, 255),stop:1 rgba(50, 50, 50, 255));
	border: 1px solid #000;
    color: #fff;

}


/*-----QTreeWidget-----*/
QTreeView
{
	show-decoration-selected: 1;
	alternate-background-color: #3a3a3a;
	selection-color: #fff;
	background-color: #2d2d2d;
	border: 1px solid gray;
	padding-top : 5px;
	color: #fff;
	font: 8pt;

}


QTreeView::item:selected
{
	color:#fff;
	background-color: #b78620;
	border-radius: 0px;

}


QTreeView::item:!selected:hover
{
    background-color: #262626;
    border: none;
    color: white;

}


QTreeView::branch:has-children:!has-siblings:closed,
QTreeView::branch:closed:has-children:has-siblings 
{
	image: url(://tree-closed.png);

}


QTreeView::branch:open:has-children:!has-siblings,
QTreeView::branch:open:has-children:has-siblings  
{
	image: url(://tree-open.png);

}


/*-----QListView-----*/
QListView 
{
	background-color: qlineargradient(spread:repeat, x1:1, y1:0, x2:1, y2:1, stop:0 rgba(83, 83, 83, 255),stop:0.293269 rgba(81, 81, 81, 255),stop:0.634615 rgba(79, 79, 79, 255),stop:1 rgba(83, 83, 83, 255));
    border : none;
    color: white;
    show-decoration-selected: 1; 
    outline: 0;
	border: 1px solid gray;

}


QListView::disabled 
{
	background-color: #656565;
	color: #1b1b1b;
    border: 1px solid #656565;

}


QListView::item 
{
	background-color: #2d2d2d;
    padding: 1px;

}


QListView::item:alternate 
{
    background-color: #3a3a3a;

}


QListView::item:selected 
{
	background-color: #b78620;
	border: 1px solid #b78620;
	color: #fff;

}


QListView::item:selected:!active 
{
	background-color: #b78620;
	border: 1px solid #b78620;
	color: #fff;

}


QListView::item:selected:active 
{
	background-color: #b78620;
	border: 1px solid #b78620;
	color: #fff;

}


QListView::item:hover {
    background-color: #262626;
    border: none;
    color: white;

}


/*-----QCheckBox-----*/
QCheckBox
{
	background-color: transparent;
    color: lightgray;
	border: none;

}


QCheckBox::indicator
{
    background-color: #323232;
    border: 1px solid darkgray;
    width: 12px;
    height: 12px;

}


QCheckBox::indicator:checked
{
    image:url(&quot;./ressources/check.png&quot;);
	background-color: #b78620;
    border: 1px solid #3a546e;

}


QCheckBox::indicator:unchecked:hover
{
	border: 1px solid #b78620; 

}


QCheckBox::disabled
{
	color: #656565;

}


QCheckBox::indicator:disabled
{
	background-color: #656565;
	color: #656565;
    border: 1px solid #656565;

}


/*-----QRadioButton-----*/
QRadioButton 
{
	color: lightgray;
	background-color: transparent;

}


QRadioButton::indicator::unchecked:hover 
{
	background-color: lightgray;
	border: 2px solid #b78620;
	border-radius: 6px;
}


QRadioButton::indicator::checked 
{
	border: 2px solid #b78620;
	border-radius: 6px;
	background-color: rgba(183,134,32,20%);  
	width: 9px; 
	height: 9px; 

}


/*-----QSlider-----*/
QSlider::groove:horizontal 
{
	background-color: transparent;
	height: 3px;

}


QSlider::sub-page:horizontal 
{
	background-color: #b78620;

}


QSlider::add-page:horizontal 
{
	background-color: #131313;

}


QSlider::handle:horizontal 
{
	background-color: #b78620;
	width: 14px;
	margin-top: -6px;
	margin-bottom: -6px;
	border-radius: 6px;

}


QSlider::handle:horizontal:hover 
{
	background-color: #d89e25;
	border-radius: 6px;

}


QSlider::sub-page:horizontal:disabled 
{
	background-color: #bbb;
	border-color: #999;

}


QSlider::add-page:horizontal:disabled 
{
	background-color: #eee;
	border-color: #999;

}


QSlider::handle:horizontal:disabled 
{
	background-color: #eee;
	border: 1px solid #aaa;
	border-radius: 3px;

}


/*-----QScrollBar-----*/
QScrollBar:horizontal
{
    border: 1px solid #222222;
    background-color: #3d3d3d;
    height: 15px;
    margin: 0px 16px 0 16px;

}


QScrollBar::handle:horizontal
{
	background-color: qlineargradient(spread:repeat, x1:1, y1:0, x2:1, y2:1, stop:0 rgba(97, 97, 97, 255),stop:1 rgba(90, 90, 90, 255));
	border: 1px solid #2d2d2d;
    min-height: 20px;

}


QScrollBar::add-line:horizontal
{
	background-color: qlineargradient(spread:repeat, x1:1, y1:0, x2:1, y2:1, stop:0 rgba(97, 97, 97, 255),stop:1 rgba(90, 90, 90, 255));
	border: 1px solid #2d2d2d;
    width: 15px;
    subcontrol-position: right;
    subcontrol-origin: margin;

}


QScrollBar::sub-line:horizontal
{
	background-color: qlineargradient(spread:repeat, x1:1, y1:0, x2:1, y2:1, stop:0 rgba(97, 97, 97, 255),stop:1 rgba(90, 90, 90, 255));
	border: 1px solid #2d2d2d;
    width: 15px;
    subcontrol-position: left;
    subcontrol-origin: margin;

}


QScrollBar::right-arrow:horizontal
{
    image: url(://arrow-right.png);
    width: 6px;
    height: 6px;

}


QScrollBar::left-arrow:horizontal
{
    image: url(://arrow-left.png);
    width: 6px;
    height: 6px;

}


QScrollBar::add-page:horizontal, QScrollBar::sub-page:horizontal
{
    background: none;

}


QScrollBar:vertical
{
    background-color: #3d3d3d;
    width: 16px;
	border: 1px solid #2d2d2d;
    margin: 16px 0px 16px 0px;

}


QScrollBar::handle:vertical
{
    background-color: qlineargradient(spread:repeat, x1:1, y1:0, x2:1, y2:1, stop:0 rgba(97, 97, 97, 255),stop:1 rgba(90, 90, 90, 255));
	border: 1px solid #2d2d2d;
    min-height: 20px;

}


QScrollBar::add-line:vertical
{
	background-color: qlineargradient(spread:repeat, x1:1, y1:0, x2:1, y2:1, stop:0 rgba(97, 97, 97, 255),stop:1 rgba(90, 90, 90, 255));
	border: 1px solid #2d2d2d;
    height: 15px;
    subcontrol-position: bottom;
    subcontrol-origin: margin;

}


QScrollBar::sub-line:vertical
{
	background-color: qlineargradient(spread:repeat, x1:1, y1:0, x2:1, y2:1, stop:0 rgba(97, 97, 97, 255),stop:1 rgba(90, 90, 90, 255));
	border: 1px solid #2d2d2d;
    height: 15px;
    subcontrol-position: top;
    subcontrol-origin: margin;

}


QScrollBar::up-arrow:vertical
{
    image: url(://arrow-up.png);
    width: 6px;
    height: 6px;

}


QScrollBar::down-arrow:vertical
{
    image: url(://arrow-down.png);
    width: 6px;
    height: 6px;

}


QScrollBar::add-page:vertical, QScrollBar::sub-page:vertical
{
    background: none;

}


/*-----QProgressBar-----*/
QProgressBar
{
    border: 1px solid #666666;
    text-align: center;
	color: #000;
	font-weight: bold;

}


QProgressBar::chunk
{
    background-color: #b78620;
    width: 30px;
    margin: 0.5px;

}
</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <widget class="QSplitter" name="splitter_2">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <widget class="QTreeWidget" name="Batch_images_tree">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>52</width>
         <height>0</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>16777215</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="baseSize">
        <size>
         <width>50</width>
         <height>0</height>
        </size>
       </property>
       <property name="toolTip">
        <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Images&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
       </property>
       <property name="sizeAdjustPolicy">
        <enum>QAbstractScrollArea::AdjustToContents</enum>
       </property>
       <attribute name="headerDefaultSectionSize">
        <number>100</number>
       </attribute>
       <column>
        <property name="text">
         <string>Image</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>Number of boxes</string>
        </property>
       </column>
      </widget>
      <widget class="QWidget" name="image_frame" native="true">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="contextMenuPolicy">
        <enum>Qt::PreventContextMenu</enum>
       </property>
       <layout class="QGridLayout" name="gridLayout_2">
        <property name="sizeConstraint">
         <enum>QLayout::SetDefaultConstraint</enum>
        </property>
        <item row="0" column="2">
         <widget class="QGraphicsView" name="graphicsView">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
            <horstretch>1</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="mouseTracking">
           <bool>true</bool>
          </property>
          <property name="sizeAdjustPolicy">
           <enum>QAbstractScrollArea::AdjustToContents</enum>
          </property>
          <property name="dragMode">
           <enum>QGraphicsView::NoDrag</enum>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QSplitter" name="splitter">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <widget class="QFrame" name="preview_frame">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>100</width>
          <height>100</height>
         </size>
        </property>
        <property name="sizeIncrement">
         <size>
          <width>0</width>
          <height>0</height>
         </size>
        </property>
        <property name="baseSize">
         <size>
          <width>0</width>
          <height>0</height>
         </size>
        </property>
        <property name="toolTip">
         <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Preview&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
        </property>
        <property name="frameShape">
         <enum>QFrame::StyledPanel</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Raised</enum>
        </property>
        <layout class="QGridLayout" name="gridLayout_3">
         <item row="0" column="0">
          <widget class="QGraphicsView" name="preview">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="sizeAdjustPolicy">
            <enum>QAbstractScrollArea::AdjustToContents</enum>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
       <widget class="QTreeWidget" name="Boxes_tree">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>0</height>
         </size>
        </property>
        <property name="toolTip">
         <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Boxes&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
        </property>
        <property name="layoutDirection">
         <enum>Qt::LeftToRight</enum>
        </property>
        <property name="sizeAdjustPolicy">
         <enum>QAbstractScrollArea::AdjustToContents</enum>
        </property>
        <column>
         <property name="text">
          <string notr="true">Box</string>
         </property>
        </column>
       </widget>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="addImage"/>
   <addaction name="actionaddBatch"/>
   <addaction name="separator"/>
   <addaction name="actionaddBox"/>
   <addaction name="separator"/>
   <addaction name="actionGenerate"/>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>520</width>
     <height>24</height>
    </rect>
   </property>
  </widget>
  <action name="addImage">
   <property name="text">
    <string>Add image</string>
   </property>
   <property name="toolTip">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Add image to selected batch&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+A, Ctrl+I</string>
   </property>
  </action>
  <action name="actionaddBatch">
   <property name="text">
    <string>Add Batch</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+A, Ctrl+B</string>
   </property>
  </action>
  <action name="actionaddBox">
   <property name="text">
    <string>add Box</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+A, Ctrl+X</string>
   </property>
  </action>
  <action name="actionGenerate">
   <property name="text">
    <string>Generate</string>
   </property>
   <property name="toolTip">
    <string>generate the Dataset</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+G</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>addImage</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>add_image()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>267</x>
     <y>211</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionaddBatch</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>add_batch()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>267</x>
     <y>211</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionaddBox</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>add_box()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>335</x>
     <y>180</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>Batch_images_tree</sender>
   <signal>clicked(QModelIndex)</signal>
   <receiver>MainWindow</receiver>
   <slot>select_Image()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>78</x>
     <y>165</y>
    </hint>
    <hint type="destinationlabel">
     <x>259</x>
     <y>145</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>Boxes_tree</sender>
   <signal>itemSelectionChanged()</signal>
   <receiver>MainWindow</receiver>
   <slot>select_box()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>421</x>
     <y>217</y>
    </hint>
    <hint type="destinationlabel">
     <x>259</x>
     <y>145</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionGenerate</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>generate()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>259</x>
     <y>145</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>add_image()</slot>
  <slot>add_batch()</slot>
  <slot>load_image()</slot>
  <slot>add_box()</slot>
  <slot>select_Image()</slot>
  <slot>select_box()</slot>
  <slot>generate()</slot>
 </slots>
</ui>
